js/popup.js

```diff
@@ -107,6 +110,12 @@ var onQmRcdMstButtonClicked = function () {
 
 }
 
+var onShowRemindersClick = function () {

```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['onShowRemindersClick' was used before it was defined.](https://www.codacy.com/app/m_3/QuantiModo-Chrome-Extension/file/1989429177/issues/source?bid=2779649&fileBranchId=3119134#l113)